src/js_parser.zig

```diff
@@ -15785,6 +15785,7 @@ fn NewParser_(
                             .binding = p.b(B.Identifier{ .ref = ref }, local.loc),
                         };
                         try partStmts.append(p.s(S.Local{ .decls = G.Decl.List.init(decls) }, local.loc));
+                        try p.declared_symbols.append(p.allocator, .{ .ref = ref, .is_top_level = true });
```

Comment:
This is used a few lines [below](https://github.com/oven-sh/bun/pull/17930/files#diff-4f250ef34adf9fcc840f98268c0903bbb96a0a0a35e1bd63a5f5f971ac441e54R15801), which is then used by the [bundler](https://github.com/jbarrieault/bun/blob/jb/issue-14585/src/bundler/bundle_v2.zig#L10159) immediately prior to renaming.